Assets/ScaleMonk Ads/Editor/ScaleMonkAdsiOSPostProcessor.cs

```diff
@@ -187,11 +187,8 @@ private static void configureSwiftBuild(string targetPath)
             "ecpz2srf59.skadnetwork",
             "gta9lk7p23.skadnetwork",
             "7953jerfzd.skadnetwork",
-            "22mmun2rn5.skadnetwork",
```

Comment:
Why are these deleted?

Reply:
It's duplicated and other ones too (238da6jt44.skadnetwork and wg4vff78zm.skadnetwork) .

Reply:
makes sense!